utils/filter.go

```diff
@@ -1,11 +1,11 @@
-package util
+package utils
 
 import (
 	"strings"
 )
 
 type (
-	//Filter representation of source filter
+	//Filter represent a filter of database, table or columns
```

Comment:
Represents a filter (which will allow to specify the data the user want to replicate or not on the database, schema, table or column level.